bin/aragorn_to_gff.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
For those scripts that were taken from ICEFinder 2 we need to add this each one:
```
Original work by [Author Name] licensed under CC BY-NC-SA 4.0
(http://creativecommons.org/licenses/by-nc-sa/4.0/)
Modified by [Your Name] on [Date]
```